### ISSUE TITLE
The Transcript should have a clearer comment loading state

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#1994]
**Description:**

Comments pop in when loaded... It'd be nice to have a better skeleton for loading states... perhaps not show events until we have comments too

https://replay.io/view?id=6f287ad3-d854-41f3-9b4d-25c3b2c7b8a2

**Comment by jonbell-lot23** [comment]
To me, this feels similar to all our other panels, where we should put a progress bar crawling across the top and show once everything is ready.

Let's sleep on it and think about it more.

**Comment by jasonLaster** [comment]
@jonbell-lot23 where are you landing on htis?

**Comment by jonbell-lot23** [comment]
Closing this out due to new comments design.
